### ISSUE TITLE
Google Chrome in IOS device compatibility issue?

### ISSUE THREAD

**Issue by jjef-man** [jamezmca/html-css-full-course#1]
Good day! 

I have been following the youtube video for this one and built my own simple webpage based on it. I just noticed this one thing that bugs me. When I view the site using Google Chrome in my IOS device, the scrolling does not end on the footer, there's still a big empty space under the footer before the scrolling stops. I thought it was only on the webpage that I made but when I visit [James' web portfolio](https://smoljames-portfolio.netlify.app) and [Brittany Chiangs'](https://brittanychiang.com) the issue is also there. This is only true using Google Chrome, I did open the webpage using Safari and the issue is not there. Was there a fix on this or is it a compatibility problem?

This is the [webpage](https://thefarmhouse.netlify.app) that I built and here is the repository [link](https://github.com/jjef-man/airBnb) for it.

**Comment by jamezmca** [comment]
Hi :)

I think it's possibly a compatibility issue - some browsers interpret code slightly differently which can be super painful.  

I have an idea of what it could be, but to help me understand can you please attach a screenshot of the behavior just so I can be sure?

Cheers,
James

**Comment by jjef-man** [comment]
Thanks for the prompt response.

Here are the screenshots I took

<img src="https://github.com/user-attachments/assets/1529f452-75c4-45ea-9247-00638cd6ba9d" width="100" height="100">

<img src="https://github.com/user-attachments/assets/e38a777d-ba8c-41d0-9946-bac30d573b85" width="100" height="100">

<img src="https://github.com/user-attachments/assets/a034267d-17cf-4fd4-8133-3c0e459d8aef" width="100" height="100">

I hope they are useful.

**Comment by jamezmca** [comment]
So I did a little research and it looks like it's an issue for chrome specifically on IOS 

(https://www.reddit.com/r/elementor/comments/15w467i/white_space_at_bottom_of_site_only_on_chrome/)

I'll look a bit more and see if I can find any other information - super annoying tho!

**Comment by jjef-man** [comment]
Yeah I know! Everything looks perfect until I scroll down to the bottom.

Though it is good to know that it has nothing to do with the coding in our end.

Thanks for your time and I hope we get a fix on this on the near future.

Cheers, 
Jeff
